src/compas/geometry/_primitives/line.py

```diff
@@ -56,6 +55,8 @@ class Line(Primitive):
     True
     """
 
+    __module__ = "compas.geometry"
```

Comment:
I think this was giving problems, so we got rid of the `__module__` redefinition recently.